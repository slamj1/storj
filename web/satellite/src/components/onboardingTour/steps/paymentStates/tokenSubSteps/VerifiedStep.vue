// Copyright (C) 2020 Storj Labs, Inc.
// See LICENSE for copying information.

<template>
    <div class="verified-step">
        <VerifiedImage/>
        <h2 class="verified-step__title">Payment Verified</h2>
        <span class="verified-step__sub-title">
            Your payment has been successfully verified. Click the button below to name your project and start uploading data.
        </span>
    </div>
</template>

<script lang="ts">
import { Component, Vue } from 'vue-property-decorator';

import VerifiedImage from '@/../static/images/onboardingTour/verified.svg';

@Component({
    components: {
        VerifiedImage,
    },
})

export default class VerifiedStep extends Vue {}
</script>

<style scoped lang="scss">
    .verified-step {
        width: 60%;
        padding-top: 30px;
        display: flex;
        flex-direction: column;
        align-items: center;
        font-family: 'font_regular', sans-serif;

        &__title {
            font-size: 18px;
            line-height: 22px;
            color: #354049;
        }

        &__sub-title {
            font-size: 12px;
            line-height: 17px;
            text-align: center;
            color: #354049;
        }
    }
</style>
