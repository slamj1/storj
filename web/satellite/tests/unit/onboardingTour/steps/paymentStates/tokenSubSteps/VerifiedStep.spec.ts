// Copyright (C) 2020 Storj Labs, Inc.
// See LICENSE for copying information.

import VerifiedStep from '@/components/onboardingTour/steps/paymentStates/tokenSubSteps/VerifiedStep.vue';

import { shallowMount } from '@vue/test-utils';

describe('VerifiedStep.vue', () => {
    it('renders correctly', (): void => {
        const wrapper = shallowMount(VerifiedStep);

        expect(wrapper).toMatchSnapshot();
    });
});
