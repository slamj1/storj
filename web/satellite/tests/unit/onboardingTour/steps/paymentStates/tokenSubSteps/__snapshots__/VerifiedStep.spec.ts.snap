// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VerifiedStep.vue renders correctly 1`] = `
<div class="verified-step">
  <verifiedimage-stub></verifiedimage-stub>
  <h2 class="verified-step__title">Payment Verified</h2> <span class="verified-step__sub-title">
        Your payment has been successfully verified. Click the button below to name your project and start uploading data.
    </span>
</div>
`;
